utils.py

```diff
@@ -22,7 +26,10 @@ def validate_request(req):
     return payload
 
 
-def is_valid_signature(x_hub_signature, data, private_key=os.environ['WEBHOOK_SECRET']):
+def is_valid_signature(
```

Comment:
formatting changes again :(

Reply:
pip's autopep8 and pep8speaks have different styles :_( I think I'll try to fix style issues at the very end now.